web/src/utils/constants.ts

```diff
@@ -17,3 +17,10 @@ export const FIRE_WEATHER_ROUTE = '/fire-weather'
 export const MORECAST_ROUTE = '/morecast'
 export const HFI_CALC_ROUTE = '/hfi-calculator'
 export const C_HAINES_ROUTE = '/c-haines'
+
+// ExpandableContainer widths
+export const PARTIAL_WIDTH = 850
+export const FULL_WIDTH = 1200
```

Comment:
I've been wondering about this - why can't we just go 100% once you've fully expanded?

Reply:
It does -- I just need to use some number because state is represented as a number

Reply:
https://github.com/bcgov/wps/pull/922/files#diff-6e18a3c8f2248cba5aab52d282913a1a4080319755220ba5fe852038282fc202R10